### PR TITLE
Import admin divs on PG12

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: thinkhazard/processing/import.py [hunk 1]
```diff
@@ -110,7 +128,7 @@ def do_execute(self):
             )
         )
 
-    def import_shapefile(self, path):
+    def import_shapefile_ogr2ogr(self, path):
```
Comment (tonio) [inline thinkhazard/processing/import.py:131]: Do you keep it for reference to get back to it once it's compatible with PG12 ?

Comment (arnaud-morvan) [reply thinkhazard/processing/import.py:131]: Yes, shp2pgsql setup take 200 Mo.
And Debian Sid has fixed GDAL 3.

**Comment by arnaud-morvan** [comment]
@tonio : I didn't manage to install a precompiled fixed GDAL.
I found easier to install PostGIS from distrib than to compile GDAL.

**Comment by tonio** [comment]
> I found easier to install PostGIS from distrib than to compile GDAL.

Sure! Good to merge then.

**Pull request merged**